### ISSUE TITLE
mysql8.0 has no autoIncrement attribute

### ISSUE THREAD

**Issue by try520** [sequelize/sequelize-auto#246]
A self-incrementing ID of type int was created in MySQL 8.0, but after sequelize-auto to model, the autoIncrement property disappeared, causing the id not to be null when inserting data

**Comment by aruntk** [comment]
until this is fixed you can use [@tkay/sequelize-auto](https://www.npmjs.com/package/@tkay/sequelize-auto)

**Comment by nramaiah** [comment]
I am having the same problem.

What is the syntax/format to include this version in my package.json. Please help.

**Comment by aruntk** [comment]
@nramaiah 

`npm install --save @tkay/sequelize-auto`

if you are using yarn

`yarn add @tkay/sequelize-auto`

**Comment by nramaiah** [comment]
@aruntk 

Not command line. How to do I reference it in package.json ?

**Comment by aruntk** [comment]
@nramaiah 

package.json
```json
dependencies: {
    "@tkay/sequelize-auto": "^0.4.31",
}
```

```js
// Usage
var SequelizeAuto = require('@tkay/sequelize-auto')
// ...
```

https://github.com/sequelize/sequelize-auto#programmatic-api

**Comment by FredLackeyOfficial** [comment]
Take a look at [`file-line-replacer`](https://github.com/FredLackey/file-line-replacer) when you have a moment.  I've had the same problem for years and cannot stand dealing with writing scripts to tweak the final files.  I created [`file-line-replacer`](https://github.com/FredLackey/file-line-replacer) to make fixing these files easier.  This allows sequelize-auto to be used, without issue, and then adjust the files when you're finished.

Here are the steps I use.  First, install it (I prefer installing it globally)...

`npm i -g file-line-replacer`

... and then, from inside of the project using sequelize, I run ... 

```
file-line-replacer \
  --search-dir "/Users/flackey/my-project/src/data/models" \
  --backup-dir "/Users/flackey/my-project/_backup" \
  --old-lines "allowNull: false,|primaryKey: true" \
  --new-lines "autoIncrement: true,|primaryKey: true" \
  --overwrite
``` 
The above command will tweak your models, adding the `autoIncrement` key, and create a backup for each file before it is modified.

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.0`
